include/linear_solvers/assemblers/assembler_eigen_implicit_linear.tcc

```diff
@@ -43,17 +43,23 @@ bool mpm::AssemblerEigenImplicitLinear<Tdim>::assemble_stiffness_matrix() {
           for (unsigned j = 0; j < nids.size(); ++j) {
             for (unsigned k = 0; k < Tdim; ++k) {
               for (unsigned l = 0; l < Tdim; ++l) {
-                stiffness_matrix_.coeffRef(
-                    nactive_node * k + global_node_indices_.at(cid)(i),
-                    nactive_node * l + global_node_indices_.at(cid)(j)) +=
-                    cell_stiffness(Tdim * i + k, Tdim * j + l);
+                if (std::abs(cell_stiffness(Tdim * i + k, Tdim * j + l)) >
+                    std::numeric_limits<double>::epsilon())
+                  tripletList.push_back(Eigen::Triplet<double>(
```

Comment:
Looks good to me, but I have a question.
When you push back the cell_stiffness to the tripletList, it seems to occur some triplets with the same row and column numbers.
Does "setFromTriplets" make the summation of all of them?